clusters/microservices-demo.yaml

```
@@ -0,0 +1,27 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: onlineboutique
+  namespace: flux-system
+spec:
+  interval: 24h
+  type: oci
+  url: oci://us-docker.pkg.dev/online-boutique-ci/charts
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: onlineboutique
```

```suggestion
  name: online-boutique
```
Nit: Could you update all the names to use kebab-case please?